numpy/testing/nosetester.py

```diff
@@ -514,3 +517,10 @@ def bench(self, label='fast', verbose=1, extra_argv=None):
         add_plugins = [Unplugger('doctest')]
 
         return nose.run(argv=argv, addplugins=add_plugins)
+
+def _numpy_tester():
+    if ".dev0" in np.__version__:
```

Comment:
this will break builds under some circumstances (try ``build_ext -i`` twice in a row), you need the ``hasattr`` check that you deleted from ``NoseTester.__init__``.